docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -109,7 +109,99 @@ var flexLayout = new FlexLayout();
+The align self overrides any option set by the parent with align items.
+
+The following figure shows how the second item works with `FlexAlignmentSelf` property:
+
+![FlexAlignmentSelf](./media/flexAlignmentSelf.png)
+
+## FlexPositionType property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexPositionType` property of an element defines how it is positioned within its parent.
+
+By default an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and then offset relative to that margin based on the values of start, end, top, and bottom.
+The offset does not affect the position of any sibling or parent elements.
+
+When positioned absolutely an element doesn't take part in the normal layout flow.
+It is instead laid out independent of its siblings. The position is determined based on the position property.
+
+The following figure shows how the second item works with `FlexPositionType` property:
+
+![FlexPositionType](./media/flexPositionType.png)
+
+## FlexAspectRatio property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexAspectRatio` property has the following properties:
+
+ - Accepts any floating point value > 0, the default is undefined.
+ - Defined as the ratio between the width and the height of a node e.g. if a node has an aspect ratio of 2 then its width is twice the size of its height.
+ - Respects the min and max dimensions of an item.
+ - Has higher priority than `FlexGrow` property.
+ - If aspect ratio, width, and height are set then the cross axis dimension is overridden.
+
+## FlexBasis property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis.
+Setting the flex basis of a child is similar to setting the width of that child if its parent is a container with flex direction: row or setting the height of a child if its parent is a container with flex direction: column.
+The flex basis of an item is the default size of that item, the size of the item before any flex grow and flex shrink calculations are performed.
+
+## FlexShrink property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflow the size of the container on the main axis.
+The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space. These two properties also work well together by allowing children to grow and shrink as needed.
+
+The flex shrink accepts any floating point value >= 0, with 1 being the default value. A container will shrink its children weighted by the child’s flex shrink value.
+
+The following figure shows how the second item works with `FlexShrink` property:
+
+![FlexShrink](./media/flexShrink.png)
+
```

Comment:
```suggestion
## FlexShrink property

The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflows the size of the container on the main axis.
The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space. These two properties also work well together by allowing children to grow and shrink as needed.

The flex shrink accepts any floating point value greater or equal to `0`. The default value is `1`. A container will shrink its children weighted by the child’s flex shrink value.
The following figure shows how the second item works with `FlexShrink` property:

![FlexShrink](./media/flexShrink.png)

> [!NOTE]
> This property is added in Tizen 6.0.

```
Can you please rephrase thes?


The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflows the size of the container on the main axis.

The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space. 


The flex shrink accepts any floating point value >= 0, with 1 being the default value. A container will shrink its children weighted by the child’s flex shrink value.

Reply:
> The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space.
> 
> A container will shrink its children weighted by the child’s flex shrink value.

Above sentences have been removed.